src/Form.js

```diff
@@ -0,0 +1,161 @@
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Email:</label>
+                      </td>
+                      <td>
+                          <input type="email" name="email" value = {email} onChange={this.handleChange} />
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Mật khẩu:</label>
+                      </td>
+                      <td>
+                          <input type="password" name="pass" value={pass} onChange={this.handleChange} />
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Avatar:</label>
+                      </td>
+                      <td>
+                          <input type="file" name="file" value = {avatar} multiple onChange={this.handleChange} />
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Số điện thoại:</label>
+                      </td>
+                      <td>
+                          <input type="number" name="phone"value = {phone} onChange={this.handleChange} />
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Tuổi:</label>
+                      </td>
+                      <td>
+                          <select name = "age" value = {age} onChange={this.handleChange} >
+                              <option name="1 tuổi" value="1 tuổi">1 tuổi</option>
+                              <option name="2 tuổi" value="2 tuổi">2 tuổi</option>
+                              <option name="3 tuổi" value="3 tuổi">3 tuổi</option>
+                              <option name="4 tuổi" value="4 tuổi">4 tuổi</option>
+                          </select >
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Giới tính:</label>
+                      </td>
+                      <td>
+                          <input type="radio" name="gender" className="input_r" value="Nam" checked={gender === "Nam"} onChange={this.handleChange}/>Nam  <br/>
+                          <input type="radio" name="gender" className="input_r" value ="Nu"checked={gender === "Nu"} onChange={this.handleChange}/> Nữ 
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>
+                          <label htmlFor="">Sở thích:</label>
+                      </td>
+                      <td>
+                          <input key={favorite.id} type="checkbox" className="input_r" value={favorite.value} name={favorite.value} checked={favorite.isChecked} onClick={this.handleChangeFavorite}/> Đá bóng <br/>
```

Comment:
`className="input_r"`
Là cái class thần thánh gì mà `radio` cũng dùng được, `checkbox` cũng dùng được? đặt tên có ý nghĩa gì?

`key={favorite.id}`: Chỉ đặt key khi nó nằm trong 1 vòng lặp
`value={favorite.value} name={favorite.value}`: `name` đọc kỹ lại `name` trong form dùng để làm gì và phải đặt như thế nào

Đã có array `favorites` ở `state` dưới này lại viết bằng tay thành 3 dòng `input`, sao không ném nó trong 1 vòng `for` rồi in cái biến đấy ở đây?